Cargo.lock

```diff
@@ -2365,6 +2357,15 @@ version = "0.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "77af24da69f9d9341038eba93a073b1fdaaa1b788221b00a69bce9e762cb32de"
 
+[[package]]
+name = "openssl-src"
+version = "111.9.0+1.1.1g"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "a2dbe10ddd1eb335aba3780eb2eaa13e1b7b441d2562fd962398740927f39ec4"
+dependencies = [
+ "cc",
+]
+
 [[package]]
 name = "openssl-sys"
```

Comment:
Is `openssl-sys` still required with this change?

Reply:
Yes, `openssl-sys` is Rust bindings to `openssl`. We still require `openssl`, but `openssl-sys` will automatically build `openssl` as opposed to requiring it to be installed on the OS.